gridt/src/app/about/tutorial/tutorial.page.html

```diff
@@ -0,0 +1,68 @@
+<ion-content>
+  <ion-grid>
+    <!-- <ion-row *ngIf="router.url === '/about'" class="ion-float-right">
+      <ion-col  class="ion-float-right">
+        <ion-button (click)="finish()">
+          <ion-icon name="close" slot="icon-only"></ion-icon>
+        </ion-button>
+      </ion-col>
+    </ion-row> -->
```

Comment:
Please do not commit commented out code. It needs to be clear if it is necessary yes or no, not half half.